src/github.sh

```diff
@@ -5,14 +5,16 @@ GITHUB_API_HEADER="Accept: application/vnd.github.v3+json"
 github::calculate_total_modifications() {
   local -r pr_number="${1}"
   local -r files_to_ignore="${2}"
+  local -r changed_file_weight="${3:-0}"
```

Comment:
existing pattern seems to be to set the default values in the actions.yml file rather than inline here